tests/test_sft_trainer.py

```diff
@@ -193,34 +150,27 @@ def test_run_causallm_pt_init_text():
     invalid_params_map,
     ids=["num_train_epochs", "grad_acc_steps"],
 )
-def test_run_causallm_pt_invalid_params(param_name, param_val, exc_msg):
+def test_run_causallm_pt_invalid_train_params(param_name, param_val, exc_msg):
     """Check if error is raised when invalid params are used to peft tune causallm models"""
     with tempfile.TemporaryDirectory() as tempdir:
-        invalid_params = copy.deepcopy(BASE_PEFT_KWARGS)
-        invalid_params["output_dir"] = tempdir
-        invalid_params[param_name] = param_val
-        model_args, data_args, training_args, tune_config = causal_lm_train_kwargs(
-            invalid_params
-        )
+        invalid_params = copy.deepcopy(TRAIN_ARGS)
+        invalid_params.output_dir = tempdir
+        setattr(invalid_params, param_name, param_val)
 
         with pytest.raises(ValueError, match=exc_msg):
-            sft_trainer.train(model_args, data_args, training_args, tune_config)
+            sft_trainer.train(MODEL_ARGS, DATA_ARGS, invalid_params, PEFT_PT_ARGS)
 
 
 def test_run_causallm_pt_with_validation():
     """Check if we can bootstrap and peft tune causallm models with validation dataset"""
     with tempfile.TemporaryDirectory() as tempdir:
-        validation_peft = copy.deepcopy(BASE_PEFT_KWARGS)
-        validation_peft["output_dir"] = tempdir
-        validation_peft["validation_data_path"] = TWITTER_COMPLAINTS_DATA
-        validation_peft["evaluation_strategy"] = "epoch"
-        model_args, data_args, training_args, tune_config = causal_lm_train_kwargs(
-            validation_peft
-        )
-
-        assert data_args.validation_data_path == TWITTER_COMPLAINTS_DATA
+        train_args = copy.deepcopy(TRAIN_ARGS)
+        train_args.output_dir = tempdir
+        train_args.evaluation_strategy = "epoch"
```

Comment:
Transformers just updated this flag deprecating `evaluation_strategy` in favor of `eval_strategy` https://github.com/huggingface/transformers/commit/60d5f8f9f04026cb801d0dc5158bf4531e250072